src/main/java/com/sateekot/codereview/service/BikeRentalService.java

```diff
@@ -0,0 +1,39 @@
+package com.sateekot.codereview.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.*;
+
+public class BikeRentalService {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BikeRentalService.class);
+
+  private String bikeBrand;
+  private int bikes_remaining;
+  private static final int DEFAULT_DURATION = 30;
+  private static final int total_no_of_bikes = 20;
+
+
+  public void rentBike() {
+    bikes_remaining--;
+    LOG.info("Remaing bikes available="+bikes_remaining);
```

Comment:
Use parameterized logging instead of string concatenation: log.info("Remaining bikes available={}", bikes_remaining);